### ISSUE TITLE
Icons disappear when TabbedPage Placed at bottom for android

### ISSUE THREAD

**Issue by pakbaz** [jsmarcus/Iconize#84]
When I use Iconize IconTabbedPage component for custom icon tabbed layout, for android specifically icons disappear when used the option of placing toolbar at the bottom

```
<iconize:IconTabbedPage
    xmlns="http://xamarin.com/schemas/2014/forms" 
    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml" 
    x:Class="Helix.PassengerPortal.Pages.MenuTabPage"
    xmlns:iconize="clr-namespace:Plugin.Iconize;assembly=Plugin.Iconize"
    xmlns:android="clr-namespace:Xamarin.Forms.PlatformConfiguration.AndroidSpecific;assembly=Xamarin.Forms.Core"
    xmlns:pages="clr-namespace:Helix.PassengerPortal.Pages"
    
    android:TabbedPage.ToolbarPlacement="Bottom"
    BarTextColor="White"
    BarBackgroundColor="{StaticResource TabBarBackground}">
    <pages:ReservationDetailsPage Title="Now" Icon="fas-location-arrow"/>
    <pages:ReservationListPage Title="Trips" Icon="fas-helicopter"/>
    <pages:ProfilePage Title="Profile" Icon="fas-user"/>
    <pages:AlertsPage Title="Alerts" Icon="fas-exclamation-triangle"/>
    
</iconize:IconTabbedPage>
```

everything works fine on iOS and when I use on android without the **android:TabbedPage.ToolbarPlacement="Bottom"** line it renders icon but at the top of the page (default for android)
I am using latest Version: 3.2.0.74

**Comment by GalaxiaGuy** [comment]
Bottom navigation is implemented quite different in Forms, and the method Iconize overrides to implement the icon doesn't get called.

This will need to be implemented first:
https://github.com/xamarin/Xamarin.Forms/issues/3397

**Comment by jsmarcus** [comment]
I think this will actually be solved once xamarin/Xamarin.Forms#3967 is released

**Comment by jsmarcus** [comment]
This is fixed in 3.4.0+
